CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Changelog
 =========
 
+* **2017-02-17**: Remove usage of `templating` component to validate template existence,
+  use twig loader instead.
```

Comment:
can we change this to "This bundle can now also directly use the Twig loader instead of the deprecated templating component. Symfony FrameworkBundle no longer requires symfony/templating since 3.2. If the templating component is available in your application, it is however still used for BC."

and please check the date, i'd guess 2017-09-25 would be correct

Reply:
I should also add the path to the bundlle ;-)

When removing it later, we should deprecate it.